tools/buildsys/src/manifest.rs

```diff
@@ -239,6 +239,15 @@ FIPS-compliant ciphers to be included in the image.
 fips = true
 ```
 
+`kmod-kits` enables a feature for users to bring their own custom kernel modules other than the
+modules provided by Bottlerocket.
+
+```ignore
+[package.metadata.build-variant.image-features]
+kmod-kits = true
+```
```

Comment:
I'd like to give this a somewhat more generic name since it will also be used to disable the inclusion of the kernel devel archive at runtime.

```suggestion
`kmod-kits` enables a feature for users to bring their own custom kernel modules other than the
modules provided by Bottlerocket.

```ignore
[package.metadata.build-variant.image-features]
external-kmod-development = true
```
```